libraries/node/network/rpc/Test.cpp

```diff
@@ -5,11 +5,11 @@
 #include <libdevcore/CommonJS.h>
```

Comment:
As I can not comment on previous files (auto generated), did you also change scripts that are creating them? please check  `jsonrpcstub.cmake` and related files. To check if files are correctly generated, I think you need to do `make jsonrpcstub` and then run that binary or something like that